lib/mongodb.js

```diff
@@ -127,6 +127,38 @@ const COMMAND_MAPPINGS = {
   estimatedDocumentCount: 'count',
 };
 
+exports.arrayToObjectReducer = arrayToObjectReducer;
```

Comment:
Let's keep `arrayToObjectReducer` as an internal implementation detail please and don't export it as public API. The less public API we have to maintain, the better for us.